viewer-prototype/src/browser/trace-server-preference.ts

```diff
@@ -0,0 +1,30 @@
+import { PreferenceSchema, PreferenceProxy, PreferenceScope } from '@theia/core/lib/browser';
+
+export const TRACE_PATH = 'trace-viewer.path';
+export const TRACE_PORT = 'trace-viewer.port';
+
+export const ServerSchema: PreferenceSchema = {
+    type: 'object',
+    properties: {
+        [TRACE_PATH]: {
+            'type': 'string',
+            'default': '',
+            'description': 'The path to trace-server executable, e.g.: /usr/bin/',
```

Comment:
e.g. should be /usr/bin/tracecompass-server since it must include the executable name

Reply:
Updated.